src/main.rs

```diff
@@ -102,6 +102,12 @@ fn main() {
         .arg(clap::Arg::with_name("cors")
              .long("cors")
              .help("Enable CORS via the \"Access-Control-Allow-Origin\" header"))
+        .arg(clap::Arg::with_name("coop")
+             .long("coop")
+             .help("Enable \"Cross-Origin-Opener-Policy\": same-origin"))
+        .arg(clap::Arg::with_name("coep")
+             .long("coep")
+             .help("Enable \"Cross-Origin-Embedder-Policy\": require-corp"))
```

Comment:
Please be consist with the help message in README.md.

Reply:
I just updated the PR with the requested changes.